src/core/routing/privateRouters.ts

```diff
@@ -0,0 +1,12 @@
+import EmployeeViewLayout from "../../pages/EmployeeView/EmployeeViewLayout";
+import Form from "../../pages/EmployeeUpdate/Form.tsx";
+import EmployeeListing from "../../pages/EmployeeListing/EmployeeListing.tsx";
+
+const privateRoute = [
+  { path: "/view-employee/:id", element: EmployeeViewLayout },
+  { path: "/edit-employee/:id", element: Form },
+  { path: "/add-employee/", element: Form },
+  { path: "/", element: EmployeeListing },
```

Comment:
FYI good to use a routing for listing in real time apps, we normally use / for some home page